frontend/docsettings.lua

```diff
@@ -150,12 +151,30 @@ function DocSettings:open(docfile)
         new.data = stored
         new.candidates = candidates
     else
-        new.data = {}
+        new.data = DocSettings:getDirectoryDefaults(docfile)
     end
 
     return setmetatable(new, {__index = DocSettings})
 end
 
+function DocSettings:getDirectoryDefaults(docfile)
+    local data = {}
+    if G_reader_settings:readSetting("directory_defaults_enabled") then
+        local defaults = G_reader_settings:readSetting("directory_defaults")
+        local base = G_reader_settings:readSetting("directory_defaults_base")
+        local directory = docfile and docfile ~= '' and docfile:match("(.*)/")
```

Comment:
Btw, our style is `"`, not `'`.

Reply:
copied from https://github.com/koreader/koreader/blob/3de3ea04ffe64f73301e9cbec305fab406530638/frontend/docsettings.lua#L32
will fix.

Reply:
There seem to be a grand total of half a dozen documents with single quotes in 'em. :-P

PS Always feel free to do `'Blabla "this is a quote" blabla'` when it's more elegant that way than escaping.